vignettes/articles/06-exploring-data.Rmd

```diff
@@ -101,18 +103,21 @@ sql.motus %>%
   select(ts, motusTagID, runLen, speciesEN, tagDepLat, tagDepLon, 
          recvDeployLat, recvDeployLon) %>% 
   collect() %>%
+  mutate(ts = as_datetime(ts)) %>% 
   summary()
 
 # same summary for the filtered sql data
 df.alltags.sub %>% 
   select(ts, motusTagID, runLen, speciesEN, tagDepLat, tagDepLon, 
-         recvLat, recvLon) %>% 
+         recvDeployLat, recvDeployLon) %>% 
+  collect() %>% 
+  mutate(ts = as_datetime(ts)) %>% 
```

Comment:
This isn't necessary because it's already timestamp format from above

Reply:
Yes, you're right! I got a little overzealous in my timestamp conversion 😅